packages/fury-adapter-oas3-parser/STATUS.md

```diff
@@ -24,6 +24,38 @@ Key:
 | tags | [✕](https://github.com/apiaryio/api-elements.js/issues/75) |
 | externalDocs | [✕](https://github.com/apiaryio/api-elements.js/issues/82) |
 
+## Info Object
+
+| title | ✓ |
+| description | ✓ |
+| termsOfService | ✓ |
+| contact | ✓ |
+| license | ✓ |
+| version | ✓ |
+
+## Contact Object
+
+| name | ✓ |
+| url | ✓ |
+| email | ✓ |
+
+## Licence Object
+
+| name | ✓ |
+| url | ✓ |
+
+## Server Object
+
+| url | ✓ |
+| description | ✓ |
+| variables | ✓ |
+
+## Server Variable Object
+
+| enum | ✓ |
```

Comment:
```suggestion
| Field Name | Support |
|:--|:--|
| enum | ✓ |
```

We're missing the table header rows on all the introduced objects here, could you update them to be consistent with the rest.

Reply:
changed